docker-compose.yml

```diff
@@ -79,16 +98,17 @@ services:
       #- MAIL_FROM_NAME=
       #- MAIL_FROM_ADDRESS=
       #- TRUSTED_PROXIES=
-      #- SKIP_PERMISSIONS_CHECKS
+      - SKIP_PERMISSIONS_CHECKS=${SKIP_PERMISSIONS_CHECKS:-false}
```

Comment:
Should this be in Lychee's `.env.example`?

Reply:
`SKIP_PERMISSIONS_CHECKS` is only used in the docker image, so it would only make sense to put in the `.env.example` of this repository (and not Lychee).

Reply:
:joy: Yup, my bad